test/images/webhook/README.md

```diff
@@ -1,45 +1,13 @@
-# Kubernetes External Admission Webhook Example
+# Kubernetes External Admission Webhook Test Image
 
-The example shows how to build and deploy an external webhook that only admits
-pods creation and update if the container images have the "grc.io" prefix.
+The image tests MutatingAdmissionWebhook and ValidatingAdmissionWebhook. After deploying
+it to kubernetes cluster. Administrator needs to create a ValidatingWebhookConfiguration
```

Comment:
After deploying it to kubernetes cluster, administrator...

Reply:
Done.